src/coord/src/coord.rs

```diff
@@ -532,6 +532,18 @@ where
                     );
                     self.sources.insert(entry.id(), frontiers);
                 }
+                CatalogItem::Table(table) => {
+                    println!("bootstrapping table: {:?}", table);
```

Comment:
intentional or debugging info?

Reply:
Sorry, this one's not meant for merging. I have to clean this up once #9659 is in because this PR partially contains changes from that other one.